### PR TITLE
docs: notes on migration to `@fluentui/react-components@9.0.0-rc.1`

### PR DESCRIPTION
**Pull request by layershifter**
This PR migration notes for `@fluentui/react-components@9.0.0-rc.1` to help customers with the upgrade.

The goal is to describe **breaking changes** and migration path for them.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.717 kB`<br />`46.859 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.479 kB`<br />`10.625 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c0d4e63ee58e60e2c6674efbacc0783cd520984e'>c0d4e63ee58e60e2c6674efbacc0783cd520984e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Avatar-mount_base.html">723</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Avatar-mount.html">711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/BaseButton-mount_base.html">822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/BaseButton-mount.html">819</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Breadcrumb-mount_base.html">2279</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Breadcrumb-mount.html">2305</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ButtonNext-mount_base.html">415</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ButtonNext-mount.html">445</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Checkbox-mount_base.html">1339</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Checkbox-mount.html">1347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/CheckboxBase-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/CheckboxBase-mount.html">1124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ChoiceGroup-mount_base.html">4145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ChoiceGroup-mount.html">4137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ComboBox-mount_base.html">922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ComboBox-mount.html">857</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/CommandBar-mount_base.html">8970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/CommandBar-mount.html">8966</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ContextualMenu-mount_base.html">7345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ContextualMenu-mount.html">7397</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DefaultButton-mount_base.html">985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DefaultButton-mount.html">997</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRow-mount_base.html">3258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRow-mount.html">3224</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRowFast-mount_base.html">3256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRowFast-mount.html">3250</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRowNoStyles-mount_base.html">3059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DetailsRowNoStyles-mount.html">3057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Dialog-mount_base.html">2253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Dialog-mount.html">2249</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DocumentCardTitle-mount_base.html">160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/DocumentCardTitle-mount.html">158</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Dropdown-mount_base.html">2765</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Dropdown-mount.html">2812</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderNext-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderNext-mount.html">1736</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-mount_base.html">156</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-mount.html">174</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-virtual-rerender.html">98</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">186</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">188</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FocusTrapZone-mount_base.html">1600</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FocusTrapZone-mount.html">1567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FocusZone-mount_base.html">1566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/FocusZone-mount.html">1540</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/IconButton-mount_base.html">1521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/IconButton-mount.html">1514</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Label-mount_base.html">325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Label-mount.html">324</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Layer-mount_base.html">2560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Layer-mount.html">2604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Link-mount_base.html">448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Link-mount.html">438</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MakeStyles-mount_base.html">1499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MakeStyles-mount.html">1458</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MenuButton-mount_base.html">1310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MenuButton-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MessageBar-mount_base.html">1742</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/MessageBar-mount.html">1741</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Nav-mount_base.html">3140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Nav-mount.html">2895</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/OverflowSet-mount_base.html">978</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/OverflowSet-mount.html">978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Panel-mount_base.html">2190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Panel-mount.html">2184</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Persona-mount_base.html">736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Persona-mount.html">741</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Pivot-mount_base.html">1251</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Pivot-mount.html">1223</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/PrimaryButton-mount_base.html">1131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/PrimaryButton-mount.html">1123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Rating-mount_base.html">6660</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Rating-mount.html">6637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SearchBox-mount_base.html">1204</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SearchBox-mount.html">1177</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Shimmer-mount_base.html">2217</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Shimmer-mount.html">2208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Slider-mount_base.html">1720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Slider-mount.html">1687</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SpinButton-mount_base.html">4303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SpinButton-mount.html">4340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Spinner-mount_base.html">407</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Spinner-mount.html">401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SplitButton-mount_base.html">2744</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SplitButton-mount.html">2753</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Stack-mount_base.html">457</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Stack-mount.html">453</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/StackWithIntrinsicChildren-mount_base.html">1968</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/StackWithIntrinsicChildren-mount.html">1977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/StackWithTextChildren-mount_base.html">4524</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/StackWithTextChildren-mount.html">4485</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SwatchColorPicker-mount_base.html">9827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/SwatchColorPicker-mount.html">9780</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TagPicker-mount_base.html">2323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TagPicker-mount.html">2327</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TeachingBubble-mount_base.html">11317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TeachingBubble-mount.html">11421</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Text-mount_base.html">389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Text-mount.html">398</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TextField-mount_base.html">1249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/TextField-mount.html">1259</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-mount_base.html">1054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-mount.html">1061</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-virtual-rerender_base.html">555</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-virtual-rerender.html">554</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Toggle-mount_base.html">712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/Toggle-mount.html">721</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/buttonNative-mount_base.html">124</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21629/perf-test/buttonNative-mount.html">136</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

File: packages/react-components/MIGRATION-NOTES.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+For more details, please check [microsoft/fluentui#20573](https://github.com/microsoft/fluentui/pull/20573).
+
+### makeStyles is Griffel now [just rename]
+
+`makeStyles` CSS-in-JS become a separate project called [Griffel](https://github.com/microsoft/griffel). It still used in Fluent UI React v9.
+
+## Typings & exports
+
+### Hooks are export with "\_unstable" suffix
```
Comment (layershifter) [inline packages/react-components/MIGRATION-NOTES.md:61]:
```suggestion
### Hooks are exported with "\_unstable" suffix
```

File: packages/react-components/MIGRATION-NOTES.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
+# Notes on migration to `@fluentui/react-components@9.0.0-rc.1`
```
Comment (Hotell) [inline packages/react-components/MIGRATION-NOTES.md:1]: can we specify migration from what version are we talking? ( based on previous convo it should be from Beta to RC)

File: packages/react-components/MIGRATION-NOTES.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+### makeStyles is now Griffel [just rename]
+
+`makeStyles` CSS-in-JS become a separate project called [Griffel](https://github.com/microsoft/griffel). It is still used in Fluent UI React v9.
+
+## Changes to the theming system
+
+### Brand ramp updated
+
+The brand colors have were updated to match the latest design.
```
Comment (khmakoto) [inline packages/react-components/MIGRATION-NOTES.md:78]:
```suggestion
The brand colors have been updated to match the latest design guidelines.
```

File: packages/react-components/MIGRATION-NOTES.md [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+- `ShorthandRenderFunction` => `SlotRenderFunction`
+- `ObjectShorthandPropsRecord` => `SlotPropsRecord`
+- `DefaultObjectShorthandProps` => `UnknownSlotProps`
+
+## Component changes
+
+### `Accordion`
+
+- `AccordionHeaderExpandIcon` has been removed and replaced by `ChevronRightRegular` from `@fluentui/react-icons`.
+- `AccordionHeader` props `children` is no longer a slot [#21285](https://github.com/microsoft/fluentui/pull/21285)
```
Comment (khmakoto) [inline packages/react-components/MIGRATION-NOTES.md:259]:
```suggestion
- `AccordionHeader` props `children` is no longer a slot. [#21285](https://github.com/microsoft/fluentui/pull/21285)
```

**Review by khmakoto — APPROVED** [review]

File: packages/react-components/MIGRATION-NOTES.md (hunk 2 above)
Comment (ling1726) [reply packages/react-components/MIGRATION-NOTES.md]:
```suggestion
# Migration notes from `@fluentui/react-components@9.0.0-beta.5`to `@fluentui/react-components@9.0.0-rc.1`
```

File: packages/react-components/MIGRATION-NOTES.md [hunk 5]
```diff
@@ -0,0 +1,407 @@
+# Notes on migration to `@fluentui/react-components@9.0.0-rc.1` from `9.0.0-beta.5`
```
Comment (ling1726) [inline packages/react-components/MIGRATION-NOTES.md:1]:
```suggestion
# Migration notes from `@fluentui/react-components@9.0.0-beta.5`to `@fluentui/react-components@9.0.0-rc.1`
```

**Review by ling1726 — APPROVED** [review]

**Review by JustSlone — APPROVED** [review]

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**